src/apps/chifra/pkg/validate/blockvalidate.go

```diff
@@ -32,7 +32,9 @@ func IsBlockHash(str string) bool {
 	return true
 }
 
-func IsBlockNumber(str string) bool {
+type blockNum = uint32
```

Comment:
I've been thinking we should have this so I can replace all occurrences of uint32 with this. Nice. I think it might want to be in a 'types' package, but don't do that now so the merge into my current branch is easier.